src/main/java/com/o19s/grandcentral/kubernetes/PodManager.java

```diff
@@ -343,7 +344,7 @@ else if (left.getLastRequest() < right.getLastRequest())
    * @throws IOException
    */
   private void refreshPods() throws IOException {
-    HttpGet podsGet = new HttpGet("https://" + k8sConfiguration.getMasterIp() + ":443/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
+    HttpGet podsGet = new HttpGet(k8sConfiguration.getProtocol() + "://" + k8sConfiguration.getMasterIp() + "/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
```

Comment:
We could drop the protocol and master ip and just have a URL.

```yaml
master_url: https://k8s.example.com
# or
master_url: http://k8s.example.com
```

I'm fairly certain I just went with master IP since that is what the GKE console displayed. It's fairly straightforward to reason out.